support-frontend/assets/pages/paper-subscription-landing/components/hero/hero.tsx

```diff
@@ -72,29 +69,25 @@ const heroParagraph = css`
 		margin-bottom: ${space[9]}px;
 	}
 `;
-const mobileLineBreak = css`
-	${from.tablet} {
+const desktopToWideLineBreak = css`
```

Comment:
Used to minimise the hero height based upon title supplied, this may be subject to change.